openapi-to-har.js

```diff
@@ -491,6 +491,9 @@ const parseParametersToQuery = function (
 
   for (let i in parameters) {
     let param = parameters[i];
+    if (param['in'] === 'query' && param['required'] === false) {
```

Comment:
It looks like the convention is to use object syntax unless the key contains a special char.

```suggestion
    if (param.in === 'query' && param.required === false) {
```

Reply:
Ah... yes, you are right